cdatgui/console/console_widget.py

```diff
@@ -8,7 +8,7 @@
 from qtconsole.rich_jupyter_widget import RichJupyterWidget
 
 from cdatgui.cdat.metadata import FileMetadataWrapper, VariableMetadataWrapper
-from cdatgui.variables import get_variables
+from cdatgui.variables import get_variables, reserved_words
```

Comment:
(by which I mean, create a new file in the root directory called constants.py and have it be there)